modules/algorithm/claws/algorithm/constexpr_algorithm.hpp

```diff
@@ -13,4 +15,16 @@ namespace claws
       }
     return out;
   }
+
+  template<class input_it, class output_it>
+  constexpr output_it move(input_it begin, input_it end, output_it out)
```

Comment:
missing noexcept specification

Reply:
The standard algorithm (even the constexpr version from C++20) is not `noexcept` since move assignment is allowed to throw

Reply:
It could easily be conditionally noexcept.

Reply:
Yes, but then `claws::move` wouldn't be a drop-in replacement for `std::move`, since `noexcept` specifications are part of the type system

Reply:
Not that we really care actually :p

Reply:
yeah I think we don't care :P